hudi-spark-datasource/hudi-spark-common/src/main/scala/org/apache/hudi/DataSourceOptions.scala

```diff
@@ -538,6 +538,8 @@ object DataSourceWriteOptions {
 
   val RECONCILE_SCHEMA: ConfigProperty[java.lang.Boolean] = HoodieCommonConfig.RECONCILE_SCHEMA
 
+  val ADD_NULL_FOR_DELETED_COLUMNS: ConfigProperty[String] = HoodieCommonConfig.ADD_NULL_FOR_DELETED_COLUMNS
```

Comment:
may not need this if we agree to support it by default when schema evol is enabled cc @rmahindra123

Reply:
For now we will keep it turned off so that we dont break backward compatability. If some oss users are relying on failing the streams with deleted columns, then making it a default evol for deleted columns will break those pipelines. Agree default true is better, but may be after a couple of release we can make that change IMO.